src/client/java/teammates/client/scripts/DataMigrationForSanitizedDataInStudentProfileAttributes.java

```diff
@@ -0,0 +1,109 @@
+package teammates.client.scripts;
+
+import teammates.client.remoteapi.RemoteApiClient;
+import teammates.client.scripts.util.DataMigrationForSanitizedDataHelper;
+import teammates.client.scripts.util.LoopHelper;
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.logic.core.ProfilesLogic;
+import teammates.common.datatransfer.attributes.StudentProfileAttributes;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Script to desanitize content of {@link StudentProfileAttributes} if it is sanitized.
+ * Fields {@link StudentProfileAttributes#shortName}, {@link StudentProfileAttributes#email},
+ * {@link StudentProfileAttributes#institute}, {@link StudentProfileAttributes#nationality},
+ * {@link StudentProfileAttributes#gender}, {@link StudentProfileAttributes#moreInfo}
+ * are no longer sanitized before saving and these fields are expected to be in their unsanitized form.
+ * This script desanitizes these fields of exisiting StudentProfileAttributes if they are sanitized so that
+ * all profiles will have unsanitized values in these fields.
+ */
+public class DataMigrationForSanitizedDataInStudentProfileAttributes extends RemoteApiClient {
+    /**
+     * Will not perform updates on the datastore if true.
+     */
+    private static final boolean isPreview = true;
```

Comment:
Just `// will not perform updates on the datastore if true.` since the variable is private?